Source/ThirdParty/SDL/android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -2098,7 +2098,7 @@ public boolean onKeyPreIme (int keyCode, KeyEvent event) {
     public InputConnection onCreateInputConnection(EditorInfo outAttrs) {
         ic = new SDLInputConnection(this, true);
 
-        outAttrs.inputType = InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD;
+        outAttrs.inputType = InputType.TYPE_CLASS_TEXT;//| InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD;
```

Comment:
Seems a bit strange to remove anything with the word 'password' in it. What does this change do exactly?

Reply:
TYPE_TEXT_VARIATION_VISIBLE_PASSWORD make Android IME disable input wide byte characters.
![image](https://raw.githubusercontent.com/urho3d/Urho3D/master/bin/Data/Textures/LogoLarge.png)

Reply:
Now it makes sense. Could you kindly put the comment you have in the screenshot to the code itself. Prefix the comment with “Urho3D” as a marker. Usually, we also mark at the beginning of the source file to say “Modified for Urho3D by...”